cinema/views.py

```diff
@@ -167,6 +199,25 @@
 
         return MovieSessionSerializer
 
+    @extend_schema(
```

Comment:
The `extend_schema` decorator is correctly used to define the schema for the `list` action in `MovieSessionViewSet`, providing detailed API documentation.